src/libraries/Native/AnyOS/System.IO.Compression.Native/entrypoints.c

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <string.h>
+#include <stdint.h>
+
+// Include System.IO.Compression.Native headers
+#include "../zlib/pal_zlib.h"
+#include "../brotli/include/brotli/decode.h"
+#include "../brotli/include/brotli/encode.h"
+#include "../brotli/include/brotli/port.h"
+#include "../brotli/include/brotli/types.h"
+
+#ifndef lengthof
+#define lengthof(rg)    (int)(sizeof(rg)/sizeof(rg[0]))
```

Comment:
Reason for the cast to `int`? This is `size_t` and the for loop below seems okay to handle this.

Reply:
GCC did not like `size_t` here. MSVC was fine though...

Reply:
What does GCC does not like about it?

Reply:
We have a combination of pedantic warnings and warn-as-error which does not like mixing `int` and `size_t` when used later in a for loop.
Another solution would be using `size_t` in the loop.